frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -134,7 +134,8 @@ import CloudProfile from '@/components/CloudProfile'
 import SecretDialogWrapper from '@/dialogs/SecretDialogWrapper'
 import { required, requiredIf } from 'vuelidate/lib/validators'
 import { getValidationErrors, isOwnSecretBinding, selfTerminationDaysForSecret } from '@/utils'
-import { includesIfAvailable } from '@/utils/validators'
+import { getCostObjectSettings, projectFromProjectList } from '@/utils/projects'
```

Comment:
utils should not depend on store (not relevant for this PR, just a remark)

Reply:
yes, I will fix this in an upcoming pr (evaluate user permissions on UI)